java/core/src/java/org/apache/orc/impl/Dictionary.java

```diff
@@ -44,9 +44,22 @@
 
   /**
    * Given the position index, return the original string before being encoded.
+   * The value of the Text in the Dictionary is copied into {@code result}.
+   *
+   * @param result the holder to copy the dictionary text into
+   * @param position the position where the key was added
    */
   void getText(Text result, int position);
 
+  /**
+   * Given the position index, write the original string, before being encoded,
+   * to the OutputStream.
+   *
+   * @return the number of byte written to the stream
```

Comment:
Shall we add `@param` for `out` and `position`?

Reply:
Done. Thanks.